### PR TITLE
County parcels

### PR DESCRIPTION
**Pull request by ian-r-rose**
This includes a script used to create a county-level parcels file, along with the census tract GEOID in which each parcel is found.

The parquet includes the metadata necessary to be a geoparquet, but needn't be read that way.

### CODE REVIEW

**Comment by hunterowens** [comment]
@tiffanychu90 is testing and running this, will merge when done.

**Comment by tiffanychu90** [comment]
Build out and overwrite the original `crosswalk_parcels_tracts` (parquet). **parcel centroids forever!**
Ref: #58 

`lacounty_parcels` can be read as geoparquet, but `crosswalk_parcels_tracts` is only parquet. crosswalk has parcels for all LA County.

File: src/load_lacounty_parcels.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"""
```
Comment (ian-r-rose) [inline src/load_lacounty_parcels.py]: @tiffanychu90 is this now safe to remove?

Comment (tiffanychu90) [reply src/load_lacounty_parcels.py]: Removed in next commit

**Pull request merged**